### ISSUE TITLE
emails from SendGrid are being sent to spam

### ISSUE THREAD

**Issue by kapil1garg** [NUDelta/pair-research-meteor#60]
<img width="1728" alt="Screen Shot 2022-02-04 at 1 11 13 PM" src="https://user-images.githubusercontent.com/3069370/153162374-d5ea1954-2e8e-416b-bbd0-7b2b6658d215.png">

Emails seems to be having issues with getting sent to spam. Not sure why.

**Comment by youralien** [comment]
Send Grid's Blog has 10 tips on how to make sure your emails don't go into Spam: 
https://sendgrid.com/blog/10-tips-to-keep-email-out-of-the-spam-folder/

Send Grid also has a tool called Email Testing (they have a 3-email-tests-per-month free version), which can help you check whether your messages would go through spam checkers.  We would need to try to see if it helps to break down possible reasons why your email would pass or not pass.
https://sendgrid.com/solutions/email-marketing/email-testing/

**Comment by youralien** [comment]
But also, can we do a temp fix for the website that says "We sent an email! Please check your spam folder for if you seem to not have received the message".

**Comment by kapil1garg** [comment]
Yup that's a great idea. I think as an alert modal when someone submits the Forgot Password form (or on the invite page) would be an easy fix to start

**Comment by kapil1garg** [comment]
![screencapture-mc-sendgrid-dynamic-templates-d-bdb4a17bf1614503aa23fbb0e56dcc49-version-030ad83c-9d4a-49a2-9a62-cba9e36d500a-editor-email-testing-view-results-1644477870-spam-2022-02-10-01_35_21](https://user-images.githubusercontent.com/3069370/153359815-e958e1c7-6287-40ca-b692-4539bfef01e8.png)


here's what the testing for the password reset email looks like
